### PR TITLE
Add translate for Why Blog?

### PR DESCRIPTION
**Pull request by GNUqb114514**
Translate for _Why Blog?_.

And an FYI: As @Sajjon mentioned in #79, It's probably necessary to break lines even in Markdowns.

### CODE REVIEW

File: posts/translations/zh-hans/why-blog.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+在学习 Rust 时我一直被几个概念所困扰，所以我尝试通过写 Markdown 组织我的想法。
+我很快便有了好几个 Markdown 文件。我想， _也许可以开始写博客？_ 但我意识到，
+_不行，我的小博客永远不会像 Niko Matsakis、Saoirse Shipwreckt、
+Huon Wilson、David Tolnay、Alexis Beingessner、Daniel Keep、Carl Fredrik Samson、
+Aleksey Kladov、Amos Wegner、Pascal Hertleif、Dimitri Sabadie、
+Daniel Henry-Mantilla、
+Steve Klabnik、Jake Goulding 和 Carol Nichols 的那样出彩。_
+这个名单一点也不全面，还缺好几十个人呢！
+有好多比我聪明的多的人已经在写这一类东西了。但是，我有一个独特的优势：我很蠢。
```
Comment (lebensterben) [inline posts/translations/zh-hans/why-blog.md:13]:
```suggestion
有好多比我聪明得多的人已经在写这一类东西了。但是，我有一个独特的优势：我很蠢。
```

File: posts/translations/zh-hans/why-blog.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+## 行，但为什么要放到 GitHub 仓库里而不是建一个网站？
+
+我是一个全栈网页网络开发人员，所以我对网站 _非常挑剔_ 。
+我致力于做一个 _完美的前端_ ，但是一点东西也没写出来。
+我知道有[无数静态网页生成器](https://www.staticgen.com/)把九成以上的工作都做完了，
+但我都试了一遍却一个也不喜欢。不是因为它们不好，实际上很多都不错，
+但我是个工匠，更喜欢自己动手。这是个问题，我知道。
+
+我差点放弃了我的这个想法，但后来我看见了 [Frank McSherry 的博客](https://github.com/frankmcsherry/blog)。
+哇！我咋这么蠢，愣是没想到可以这样？不需要折腾一点前端！只要把文章放到 GitHub 上，
```
Comment (lebensterben) [inline posts/translations/zh-hans/why-blog.md:32]:
```suggestion
哇！我怎么这么蠢，愣是没想到可以这样？不需要折腾一点前端！只要把文章放到 GitHub 上，
```

File: posts/translations/zh-hans/why-blog.md (hunk 1 above)
Comment (GNUqb114514) [reply posts/translations/zh-hans/why-blog.md:13]: Oops.

**Comment by GNUqb114514** [comment]
Thanks. @lebensterben

**Pull request merged**